### PR TITLE
Add a constructor for the Curve25519Public key type

### PR DESCRIPTION
**Pull request by poljar**
I took the opportunity to rename the vodozemac module as well, since it will soon get some more functionality for the QR login support.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # UNRELEASED
 
+-   Add a constructor for the `Cruve25519PublicKey` type. This allows us to
```
Comment (Hywan) [inline CHANGELOG.md:3]:
```diff
- Cruve
+ Curve
```

**Review by Hywan — APPROVED** [review]
If there is a possibility to test this new constructor, that would be great :-].

File: CHANGELOG.md (hunk 1 above)
Comment (poljar) [reply CHANGELOG.md]: https://github.com/matrix-org/matrix-rust-sdk-crypto-wasm/pull/117/commits/7e005abc27fb6c16fbe88c46f4b83eaf7f3d5d11

**Comment by poljar** [comment]
> If there is a possibility to test this new constructor, that would be great :-].

Added a test in https://github.com/matrix-org/matrix-rust-sdk-crypto-wasm/pull/117/commits/879759a08f4ab6e6f21c24538f54fe65efe1ecf1.

**Pull request merged**